### ISSUE TITLE
Test reports at root element

### ISSUE THREAD

**Issue by wraytw** [palmerj3/jest-junit#40]
I need the test failures to report at the root element of the tree. Is there a way to report if a failure occurred on the root xml element?

Current:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite id="0" name="/app/src/Test.test.js" errors="0" package="/app/src/Test.test.js" hostname="localhost" tests="1" failures="0" time="0.192" timestamp="2018-02-02T16:16:46">
   ...
  </testsuite>
  <testsuite id="1" name="/app/src/App.test.js" errors="0" package="/app/src/App.test.js" hostname="localhost" tests="1" failures="0" time="0.353" timestamp="2018-02-02T16:16:46">
    ...
  </testsuite>
</testsuites>
```

Desired:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites failures="0">
  <testsuite id="0" name="/app/src/Test.test.js" errors="0" package="/app/src/Test.test.js" hostname="localhost" tests="1" failures="0" time="0.192" timestamp="2018-02-02T16:16:46">
   ...
  </testsuite>
  <testsuite id="1" name="/app/src/App.test.js" errors="0" package="/app/src/App.test.js" hostname="localhost" tests="1" failures="0" time="0.353" timestamp="2018-02-02T16:16:46">
    ...
  </testsuite>
</testsuites>
```

Notice the failures attribute on the first element.

Any help is appreciated.

**Comment by palmerj3** [comment]
@wraytw that seems like something I could add this weekend. I have been working off the IBM JUnit spec and I do see the "failures" attribute is valid in the root "testsuites" object so this shouldn't break anyone.

https://www.ibm.com/support/knowledgecenter/en/SSQ2R2_9.5.0/com.ibm.rsar.analysis.codereview.cobol.doc/topics/cac_useresults_junit.html

**Comment by palmerj3** [comment]
@wraytw I published 3.5.0 which contains this addition. Test it out and let me know how it goes!

https://github.com/palmerj3/jest-junit/releases/tag/v3.5.0

**Comment by wraytw** [comment]
You sir, are a great human. Thank you!
